examples/derivative.cpp

```diff
@@ -34,11 +32,11 @@ int main(int argc, char **argv) {
     // Setting up analytic functions
     const auto alpha = 3.0;
     const auto r_0 = mrcpp::pi - 3.0;
-    auto f = [alpha, r_0] (const double *r) -> double {
+    auto f = [alpha, r_0] (const mrcpp::Coord<D> &r) -> double {
         auto R = fabs(r[0] - r_0);
         return exp(-alpha*R);
     };
-    auto df = [alpha, r_0] (const double *r) -> double {
+    auto df = [alpha, r_0] (const mrcpp::Coord<D> &r) -> double {
         auto R = fabs(r[0] - r_0);
```

Comment:
Nitpick, but replacing `fabs` with `std:abs` will work for any numerical type, not only floats/

Reply:
Yes, I'm replacing them whenever I see them. I guess I missed this one